vmdk_plugin/Makefile

```diff
@@ -350,7 +350,7 @@ test-all:  test e2e-test testremote
 # An end-to-end test target for vSphere Docker Volume Driver
 e2e-test:
 	$(log_target)
-	$(SCRIPTS)/build.sh e2e-dkrVolDriver-test "$(ESX)" "$(VM1)" "$(VM2)"
+	$(SCRIPTS)/build.sh e2e-dkrVolDriver-test "$(ESX)" "$(VM1)" "$(VM2)" "$(DS1)" "$(DS2)" "$(CNAME)" "$(T1)"
```

Comment:
Yeah we can do that. Aren't we going to add an extra overhead for any code contributor? 

IMO, the same we can achieve through testMain() that you have introduced before running any tests to keep it consistent by using go program only.  

Other approach I am thinking of maintaining consistency, it is better to have "on demand" queries instead of having a scripts which prepares the environment before hand, this approach is simpler while dealing with CI too. Only specific tests/testgroups invokes govc queries.

Going forward, we will be breaking down tests into separate areas (e.g. swarm, vmgroup, basic_test, plugin/refcount tests etc); with the minimal ask from user test would be able to run instead of preparing all possible environment variables.